### PR TITLE
AWS marketplace image updated to AWSLinux2

### PR DESCRIPTION
**Pull request by alephnull**
- update location of scripts
- move to systemd

### CODE REVIEW

**Review by marksou — APPROVED** [review]

File: tyk-docs/content/get-started/with-tyk-multi-cloud/tutorials/installation-on-aws.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ This guide assumes the "1-Click" install was selected on the Marketplace and an
 ### To setup a Tyk Hybrid Gateway instance provisioned by the "1-Click" option
 
 1. Connect to the instance using SSH (make sure to use the key specified during the provisioning), e.g. `ssh -i your_key.pem ec2-user@instance.address`
-2. Once inside the instance execute `sudo /opt/tyk-gateway/setup_hybrid.sh -o your-organisation-id -k your-api-key` (more options described below)
+2. Once inside the instance execute `sudo /tmp/setup_hybrid.sh -o your-organisation-id -k your-api-key` (more options described below)
```
Comment (excieve) [inline tyk-docs/content/get-started/with-tyk-multi-cloud/tutorials/installation-on-aws.md]: Is this still correct? This shouldn't be in `/tmp` I think.

Comment (alephnull) [reply tyk-docs/content/get-started/with-tyk-multi-cloud/tutorials/installation-on-aws.md]: Done.

**Review by excieve — APPROVED** [review]
LGTM thanks!

**Comment by marksou** [comment]
@alephnull Did we not change this in https://github.com/TykTechnologies/tyk-docs/commit/b02c185b80a3481c482dfa22680337b6843bf57d

**Comment by alephnull** [comment]
This largely reverts those changes and a small update.

**Pull request closed without merging**